internal/lang/eval.go

```diff
@@ -305,19 +307,26 @@ func (s *Scope) EvalReference(ref *addrs.Reference, wantType cty.Type) (cty.Valu
 // Most callers should prefer to use the evaluation helper methods that
 // this type offers, but this is here for less common situations where the
 // caller will handle the evaluation calls itself.
-func (s *Scope) EvalContext(refs []*addrs.Reference) (*hcl.EvalContext, tfdiags.Diagnostics) {
-	return s.evalContext(nil, refs, s.SelfAddr)
+//
+// The [context.Context] passed to this function is the one that will be used
+// when making requests to providers if any of the references are to
+// provider-contributed functions, and so callers should typically use
```

Comment:
So far, we've always referred to these functions as "provider-defined functions".  If that terminology does not match the SDK, we should adopt what ever exists there.